WalletWasabi.Fluent/Views/Shell/TitleBarView.axaml

```diff
@@ -65,13 +76,20 @@
         </Grid.ColumnDefinitions>
 
         <ContentControl ContentTemplate="{StaticResource WasabiLogo}" />
-        <ContentControl Grid.Column="1" Grid.ColumnSpan="2" ContentTemplate="{StaticResource Title}" />
+
+        <WrapPanel Grid.Column="1" Grid.ColumnSpan="2"  VerticalAlignment="Center">
+          <ContentControl ContentTemplate="{StaticResource Title}" />
+          <Border Classes="NetworkBadge">
+            <TextBlock>TestNet</TextBlock>
+          </Border>
+        </WrapPanel>
+
         <ContentControl Grid.Column="3" ContentTemplate="{StaticResource SearchBar}" />
       </Grid>
     </DataTemplate>
 
     <DataTemplate x:Key="MacOSLayout">
-      <Grid>
+      <Grid DataContext="{Binding $parent.DataContext}">
```

Comment:
@SuperJMN i wonder if there's a cleaner way of doing this?

Reply:
@jmacato Sorry for the late reply. I'm afraid there's no better method for now. I was surprised that Bindings didn't work as expected in this situation when I wrote the `TitleBarView`. It would be nice if we had a discussion about this with the Avalonia team to see if it's a bug or not.